front/app/services/appConfiguration.ts

```diff
@@ -94,6 +94,11 @@ export interface IAppConfigurationSettings {
     identifier: string;
     secret: string;
   };
+  vienna_login?: {
+    allowed: boolean;
+    enabled: boolean;
+    environment: string;
+  };
```

Comment:
This should actually be added via the module. For some reason, other logins aren't, but they also should I think.

Reply:
How do I do this?

Reply:
You can see an example in \front\app\modules\free\user_confirmation\services\appConfiguration.ts